src/main/java/com/google/sps/utils/QuizTimingPropertiesUtils.java

```diff
@@ -18,6 +18,41 @@
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.PreparedQuery;
 import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.FetchOptions;
+import java.util.Date;
+import java.text.DateFormat;
+import java.util.Random;
+import java.util.logging.Logger;
+
+ 
+public final class QuizTimingPropertiesUtils {
+
+    private static final Logger log = Logger.getLogger(QuizTimingPropertiesUtils.class.getName());
+
+    List<String> quiz_questions = new ArrayList<String>(List.of(
+        "Which plant has the most food growing from it?",
+        "Which plant has the prettiest colors?",
+        "Which plant is likely to grow the fastest?",
+        "Which plant is likely to require the most maintenance to take care of?",
+        "Which plant is most likely to be poisonous?",
+        "Which plant is most likely to survive really dry weather?",
+        "Which plant is the largest?",
+        "Which plant looks edible?",
+        "Which plant looks the healthiest?",
+        "Which plant looks the most serene?",
+        "Which plant needs the most sunlight?",
+        "Which plant needs the most water?",
+        "Which plant represents you?",
+        "Which plant represents your best friend?",
+        "Which plant will most likely impress your friends and family?",
+        "Which plant would look the best inside as a houseplant?",
+        "Which plant would look the best outside in a garden?",
+        "Which plant would you give as a gift?"
+    ));
+
 import java.util.List;
 import com.google.appengine.api.datastore.FetchOptions;
 import com.google.appengine.api.datastore.Query.Filter;
```

Comment:
u might wanna move these imports so they're before the class declaration

Reply:
That's weird, these imports were suppose to be deleted...let me know if you still see them in the new commit